frontend/src/app/commonComponents/MultiSelect/MultiSelect.tsx

```diff
@@ -39,18 +41,24 @@ type PillProps = {
 const Pill = (props: PillProps) => (
   <div className="pill">
     {props.option.label}
-    <div
-      className="close-button"
-      data-testid={`${props.option.label}-pill-delete`}
+    <Button
+      className="close-button usa-button--unstyled"
       onClick={() => props.onDelete(props.option)}
     >
-      <FontAwesomeIcon fontSize={24} icon={"times"} />
-    </div>
+      <FontAwesomeIcon
+        aria-hidden={false}
+        fontSize={24}
+        icon={"times"}
+        aria-label={"Delete device"}
```

Comment:
I think this aria-label should probably be a prop on the multiSelect, since the icon's purpose won't always be device deletion. "Delete `label`" maybe?

Reply:
ooh thank you! good catch

Reply:
fixed in the latest commit!